Examples/Python/python/acts/examples/__init__.py

```diff
@@ -285,3 +286,43 @@ def _allmods(mod, base, found):
     if not quiet and donemods:
         print("dump_args for module functions:", ", ".join(donemods))
     return alldone
+
+
+class CustomLogLevel(Protocol):
+    def __call__(
+        self,
+        override: Optional[acts.logging.Level] = None,
+        min: acts.logging.Level = acts.logging.VERBOSE,
+        max: acts.logging.Level = acts.logging.FATAL,
```

Comment:
```suggestion
        minLevel: acts.logging.Level = acts.logging.VERBOSE,
        maxLevel: acts.logging.Level = acts.logging.FATAL,
```
I don't know if the _self_ should be in there. Is that a feature of how this is defined?

Reply:
I am not sure about this but this is how I found it on the web. it kind of made sense because the callable might use some state in the general case